src/settings/v2/components/QASettings.tsx

```diff
@@ -49,62 +36,47 @@ const QASettings: React.FC<QASettingsProps> = ({ indexVaultToVectorStore }) => {
             description={
               <div className="flex items-center gap-1.5">
                 <span className="leading-none">Decide when you want the vault to be indexed.</span>
-                <Popover
-                  open={openPopoverIds.has("index-help")}
-                  onOpenChange={(open) => {
-                    if (open) {
-                      handlePopoverOpen("index-help");
-                    } else {
-                      handlePopoverClose("index-help");
-                    }
-                  }}
-                >
-                  <PopoverTrigger asChild>
-                    <HelpCircle
-                      className="h-5 w-5 sm:h-4 sm:w-4 cursor-pointer text-muted hover:text-accent translate-y-[1px]"
-                      onMouseEnter={() => handlePopoverOpen("index-help")}
-                      onMouseLeave={() => handlePopoverClose("index-help")}
-                    />
-                  </PopoverTrigger>
-                  <PopoverContent
-                    container={modalContainer}
-                    className="w-[90vw] max-w-[400px] p-2 sm:p-3"
-                    side="bottom"
-                    align="center"
-                    sideOffset={0}
-                    onMouseEnter={() => handlePopoverOpen("index-help")}
-                    onMouseLeave={() => handlePopoverClose("index-help")}
-                  >
-                    <div className="space-y-2 sm:space-y-2.5">
-                      <div className="rounded bg-callout-warning/10 p-1.5 sm:p-2 ring-ring">
-                        <p className="text-callout-warning text-xs sm:text-sm">
+                <TooltipProvider delayDuration={0}>
```

Comment:
This tooltip used the wrong component and looked pretty odd. Cleaned it up as I'm updating QA settings.